perfmetrics/scripts/fetch_metrics.py

```diff
@@ -51,15 +73,21 @@ def _parse_arguments(argv):
 
   args = _parse_arguments(argv)
 
-  if args.upload:
-    temp = fio_metrics_obj.get_metrics(args.fio_json_output_path, FIO_WORKSHEET_NAME)
-  else:
-    temp = fio_metrics_obj.get_metrics(args.fio_json_output_path)
+  bigquery_obj = bigquery.ExperimentsGCSFuseBQ(constants.PROJECT_ID, constants.DATASET_ID)
+
+  temp = fio_metrics_obj.get_metrics(args.fio_json_output_path)
+  metrics_data = fio_metrics_obj.get_values_to_export(temp)
+  if args.upload_gs:
+    fio_metrics_obj.upload_metrics_to_gsheet(metrics_data, FIO_WORKSHEET_NAME)
+  if args.upload_bq:
+    if not args.config_id or args.start_time_build:
```

Comment:
would this go into the if condition if config_id is passed but start_time_build is not passed ?

Reply:
Yes, I'll correct it. Thanks for pointing this out!